packages/utils/typescript/lib/generators/common/models/attributes.js

```diff
@@ -114,21 +114,21 @@ const getAttributeModifiers = (attribute) => {
       throw new Error('typeof min/max values mismatch');
     }
 
-    const typeofMinMax = (max && typeofMax) ?? (min && typeofMin);
     let typeKeyword;
 
-    // Determines type keyword (string/number) based on min/max options, throws error for invalid types
-    switch (typeofMinMax) {
-      case 'string':
-        typeKeyword = ts.SyntaxKind.StringKeyword;
-        break;
-      case 'number':
-        typeKeyword = ts.SyntaxKind.NumberKeyword;
-        break;
-      default:
-        throw new Error(
-          `Invalid data type for min/max options. Must be string or number, but found ${typeofMinMax}`
-        );
+    // use 'string'
+    if (typeofMin === 'string' || typeofMax === 'string') {
```

Comment:
What if the types differ?

Reply:
[They can't](https://github.com/strapi/strapi/pull/19490/files/c9a57597c7b072971eb34a989622cae8c70135ed#diff-ededeca63a5cb676ae858edc8ae032d94abbed72806b9a4a32e7ca5189b588bfR113)